### DISCUSSION TITLE
Help needed with Debug

### DISCUSSION THREAD

**Discussion by gonardfreeman** [mrcjkb/rustaceanvim#473] [Q&A] [answered]
### Feature description

Greetings,
I'm struggling on launching debug session.
When I'm typing `RustLsp debug` or `RustLsp debuggables` I'm getting this error: Debug adapter is disabled.
I haven't added any setup and installed `lldb-dap` with help of brew (brew install llvm), but never added anything to path.
I'm using M1 Mac as my system.
Exact steps of what I'm doing:
- open `main.rs`
- toggle break point with `:DapToggleBreakpoint`
- execute `:RustLsp debug`
Could someone please help of what I'm missing, because I feel it is something very simple

**Comment by mrcjkb** [comment]
Hey :wave: 

Please don't use "feature request" issues for support requests :sweat_smile: 

> execute `:RustLsp debug`

There's no `:RustLsp debug` command. You want to stick with `:RustLsp debuggables`.

> `Debug adapter is disabled.`

`:checkhealth rustaceanvim` will likely tell you what the problem is.

See also: [README.md#Troubleshooting](https://github.com/mrcjkb/rustaceanvim?tab=readme-ov-file#stethoscope-troubleshooting)

**Reply by gonardfreeman** [reply]
Sorry, but there was only bug and feature available to select 😅

**Reply by mrcjkb** [reply]
You can use [discussions](https://github.com/mrcjkb/rustaceanvim/discussions/new?category=q-a) for general questions (it can also be good to search existing ones) :smile:

**Reply by mrcjkb** [reply, marked as answer]
Your health check output does not list any debug adapters. So it's not visible on the path.

Note: There appears to be a bug in the health check preventing it from showing a warning.

**Reply by gonardfreeman** [reply]
thank you!
I'll dig deeper

**Reply by gonardfreeman** [reply]
OMG!
it is working!
I should've add `lldb-dap` separately from llvm... despite that fact that in terminal it was executing normally.

**Comment by gonardfreeman** [comment]
here check health results. Everything looks good.
Mb I should run lldb on background or something? because run command works fine
<img width="1482" alt="Screenshot 2024-08-05 at 12 45 24" src="https://github.com/user-attachments/assets/85db6c3d-5d6a-4a3e-b194-c15b37db1891">
